src/components/Gallery.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Movie from './Movie';
+import { getMovies } from '../util/swapi';
+import Header from './Header';
+import './Gallery.css';
+
+class Gallery extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            isLoading: true,
+            movieDetails: null
+        };
+    }
+
+    componentDidMount() {
+        getMovies()
+            .then(movieDetails => {
+                this.setState({ movieDetails });
+            })
+            .catch(() => {
+                this.setState({ movieDetails: { count: 0 } });
+            })
+            .finally(() => {
+                this.setState({ isLoading: false });
+            });
+    }
+
+    makeMovies() {
+        let swMovies = [];
```

Comment:
Small detail: when a variable is not reassigned we should use const instead of let.

Reply:
done